products/models.py

```diff
@@ -0,0 +1,71 @@
+from django.db import models
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=40, unique=True)
+    is_active = models.BooleanField(default=True)
+    creation_date = models.DateTimeField(auto_now_add=True)
+
+    @classmethod
+    def get_default_pk(cls):
+        no_category, created = cls.objects.get_or_create(
+            name="No Category",
```

Comment:
I dont think we need this default. as you already have allowed null category in product then it didn't have any purpose.